makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: build test clean docker
+.PHONY: build build-nats test clean docker docker-nats vendor
```

Comment:
What are these new PHONY targets used for? `build-nats` `docker-nats`

Reply:
For got to pull them out.

Reply:
Removed those and add tidy